### PR TITLE
Feature/cdap 14447 fix checkstyle

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

File: kafka-plugins-0.10/src/main/java/co/cask/hydrator/plugin/batch/source/KafkaBatchSource.java [hunk 1]
```diff
@@ -97,14 +97,17 @@
...
     @Macro
     @Nullable
     private String partitions;
 
-    @Description("The initial offset for each topic partition. This offset will only be used for the" +
-      "first run of the pipeline. Any subsequent run will read from the latest offset from previous run." +
-      "Offsets are inclusive. If an offset of 5 is used, the message at offset 5 will be read.")
+    @Description("The initial offset for each topic partition in partition1:offset1,partition2:offset2 form. " +
+      "These offsets will only be used for the first run of the pipeline. " +
+      "Any subsequent run will read from the latest offset from previous run." +
```
Comment (albertshau) [inline kafka-plugins-0.10/src/main/java/co/cask/hydrator/plugin/batch/source/KafkaBatchSource.java]: missing space after period

File: kafka-plugins-0.8/src/main/java/co/cask/hydrator/plugin/batch/source/KafkaBatchSource.java [hunk 2]
```diff
@@ -95,14 +94,17 @@
...
     @Macro
     @Nullable
     private String partitions;
 
-    @Description("The initial offset for each topic partition. This offset will only be used for the" +
-      "first run of the pipeline. Any subsequent run will read from the latest offset from previous run." +
-      "Offsets are inclusive. If an offset of 5 is used, the message at offset 5 will be read.")
+    @Description("The initial offset for each topic partition in partition1:offset1,partition2:offset2 form. " +
+      "These offsets will only be used for the first run of the pipeline. " +
+      "Any subsequent run will read from the latest offset from previous run." +
```
Comment (albertshau) [inline kafka-plugins-0.8/src/main/java/co/cask/hydrator/plugin/batch/source/KafkaBatchSource.java]: missing space after period

File: kafka-plugins-0.10/src/main/java/co/cask/hydrator/plugin/batch/source/KafkaBatchSource.java (hunk 1 above)
Comment (chtyim) [reply kafka-plugins-0.10/src/main/java/co/cask/hydrator/plugin/batch/source/KafkaBatchSource.java]: fixed

File: kafka-plugins-0.8/src/main/java/co/cask/hydrator/plugin/batch/source/KafkaBatchSource.java (hunk 2 above)
Comment (chtyim) [reply kafka-plugins-0.8/src/main/java/co/cask/hydrator/plugin/batch/source/KafkaBatchSource.java]: fixed

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**